### ISSUE TITLE
Unable to register with ProtonMail email.

### ISSUE THREAD

**Issue by oqkr** [codenvy/codenvy#2753]
I am unable to register using the email address `oqkr@protonmail.com`.

Why has this happened, and what can I do to fix it?

If I cannot use my email address, I will be unable to try this service: The application asks for too many and too broad GitHub permissions for me to register using my GitHub account.

**Comment by oqkr** [comment]
Why I was hesitant to register with GitHub:

![This is a bit much.](https://i.gyazo.com/7de9ae895f594bc872a3a802d4dd6346.png)

**Comment by elliottshort** [comment]
I agree - this permission set doesn't make a whole lot of sense given what it does...

**Comment by leonjmac** [comment]
Second this one - when I try to register, it simply says "User mail xxxxxx@protonmail.ch is forbidden". Moreover, when I attempt to link using my GitHub account (permissions have now been revoked), it generates an error message too. Has anyone with a ProtonMail account had any joy?
